paperweight-lib/src/main/kotlin/tasks/CreatePaperclipJar.kt

```diff
@@ -104,7 +104,7 @@ abstract class CreatePaperclipJar : JavaLauncherZippedTask() {
         val originalJar = originalBundlerJar.path
         val vanillaSha256Hash = originalJar.sha256asHex()
         val vanillaSha1Hash = toHex(originalJar.hashFile(digestSha1))
-        val vanillaUrl = "https://launcher.mojang.com/v1/objects/$vanillaSha1Hash/server.jar"
+        val vanillaUrl = "https://piston-data.mojang.com/v1/objects/$vanillaSha1Hash/server.jar"
```

Comment:
this should maybe use the URL from the version manifest similar to the bundler download task in InitialTasks

Reply:
This can possibly be addressed at a later date